repos/system_upgrade/common/actors/rpmscanner/libraries/rpmscanner.py

```diff
@@ -25,6 +25,8 @@
 
 def _get_package_repository_data_yum():
     yum_base = yum.YumBase()
+    # DNF configuration is not loaded here, since no impact for operations
+    # done by the actor is observed here
```

Comment:
this was meant for `_get_package_repository_data_dnf()` function :) but that's a nitpick. no need to change it